lttoolbox/alphabet.cc

```diff
@@ -240,7 +239,7 @@ Alphabet::getSymbol(wstring &result, int const symbol, bool uppercase) const
   }
   else if(symbol >= 0)
   {
-    result += static_cast<wchar_t>(towupper(static_cast<wint_t>(symbol)));
+    result += static_cast<UChar>(toupper(static_cast<wint_t>(symbol)));
```

Comment:
`wint_t` and plain C `toupper()` looks wrong.

Reply:
yeah, didn't finish converting that part